apps/avatax/scripts/deploy.ts

```diff
@@ -0,0 +1,13 @@
+import { default as envUtils } from "@next/env";
+import { execSync } from "child_process";
+
+envUtils.loadEnvConfig(".");
```

Comment:
I cant find info about this package. Why do we need this? We likely use dotenv in other places

Reply:
Removed in favour of dotenv